### ISSUE TITLE
Make "Auth, Headers, Body, e.t.c." tabs collapsable

### ISSUE THREAD

**Issue by chimit** [a85/POSTMan-Chrome-Extension#3985]
## App Details:
```
Postman for Mac
Version 5.5.0
OS X 17.3.0 / x64
```
For example, for `GET` requests there is no big need to keep Auth tab always opened, while it takes a lot of space:

![postman 2017-12-18 18-21-16](https://user-images.githubusercontent.com/839349/34101450-c036a3a4-e420-11e7-916c-e3fcfcac6742.png)

If you make these tabs collapsable it would be much better for `GET` requests and not worse for all other types of requests.

**Comment by sdnts** [comment]
Hi @chimit, thanks for the suggestion! We will look into this :)

**Comment by pedzed** [comment]
This little thing becomes so extremely frustrating. Every single time you have to scroll down. 

What I currently do is switch to the Headers tab, as it is the most compact tab. But it still is not the same as being able to collapse it.

Related: https://github.com/postmanlabs/postman-app-support/issues/4872

**Comment by vkaegis** [comment]
Hey @pedzed! Apologies for the long wait on this one, but we're working on a whole bunch of improvements around tabs and how they represent information. Expect to see things like collapsible panes in the coming versions.

**Comment by DannyDainton** [comment]
Collapsible panes are not part of the product.

When in the 2 pane mode, those request menu options can be minimised if not needed:

<img width="1355" alt="image" src="https://user-images.githubusercontent.com/17089546/177379989-a95c87fc-bc73-4b0e-b8ee-ab68a794d89d.png">
